config/application.yml.example

```diff
@@ -21,3 +21,5 @@ CURRENCY: AUD
 #
 # DISCOURSE_URL must be the URL of your Discourse instance.
 #DISCOURSE_URL: "https://noticeboard.openfoodnetwork.org.au"
+
+GOOGLE_MAPS_API_KEY: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
```

Comment:
A comment would be nice so that people know what the key is for and where to get it from.